modules/P05_firmware_bin_extractor.sh

```diff
@@ -208,7 +206,7 @@ binwalking() {
 
   export OUTPUT_DIR_binwalk
   OUTPUT_DIR_binwalk=$(basename "$FIRMWARE_PATH")
-  OUTPUT_DIR_binwalk="$LOG_DIR"/extractor/"$OUTPUT_DIR_binwalk"_binwalk_emba
+  OUTPUT_DIR_binwalk="$FIRMWARE_PATH_CP""/""$OUTPUT_DIR_binwalk"_binwalk_emba
 
   echo
   print_output "[*] Extracting firmware to directory $OUTPUT_DIR_binwalk"
```

Comment:
we have to adjust also line 235 and 238 otherwise the ipk and deb extractor will fail